authors/apps/articles/views.py

```diff
@@ -310,3 +311,91 @@ def get(self, request, slug):
             "dislikes": dislikes_count,
         }
         return Response(data=likes_dislikes, status=status.HTTP_200_OK)
+
+
+class FavoriteView(generics.CreateAPIView):
+    """This class creates a new like or dislike"""
```

Comment:
@ElMonstro, does the class create a like or dislike? I thought the PR was about favoriting.